docs/walkthrough/investigating-prometheus-alerts.md

```diff
@@ -0,0 +1,70 @@
+# Investigating Prometheus Alerts
+
+You can investigate Prometheus/AlertManager alerts using HolmesGPT by connecting it to your AlertManager instance. This allows you to run investigations on all active alerts or a specific alert and optionally send the results to a Slack channel.
+
+## Prerequisites
+
+- HolmesGPT CLI installed ([installation guide](../installation/cli-installation.md))
+- An AI provider API key configured ([setup guide](../ai-providers/index.md))
+- Access to your AlertManager instance
+
+## Investigating a Prometheus Alert Using HolmesGPT
+
+
+### Step 1: Create a Test Alert
+
+Let's create a dummy alert in Prometheus for our investigation. Run the following command to create a`KubePodCrashLooping` alert:
```

Comment:
This isn't quite right - this doesn't necessary create a Prometheus alert. It deploys the kubernetes workload which is crashing but whether there will be an alert or not depends on your alerts + it will take time. Can you deploy this workload + send an alert to alertmanager manually?